ui/app/AppLayouts/Wallet/popups/ReceiveModal.qml

```diff
@@ -30,6 +32,13 @@ StatusModal {
     property bool switchingAccounsEnabled: true
     property bool changingPreferredChainsEnabled: true
 
+    property string qrImageSource: RootStore.getQrCode(d.visibleAddress)
+    property var getNetworkShortNames: function(chainIDsString) {
+        return RootStore.getNetworkShortNames(chainIDsString)
+    }
+
+    property var networksModel: RootStore
```

Comment:
If the name is `networksModel`, could we make it point to model, not entire store?

`property var networksModel: RootStore.layer1Networks`